PROTOCOL.md

```diff
@@ -2928,6 +3128,24 @@ Examples
 }
 ```
 
+## Delta Sharing Streaming Specs
+Delta Sharing Streaming is supported starting from delta-sharing-spark 0.6.0. As it's implemented
+based on spark structured streaming, it leverages a pull model to consume the new data from the
+delta sharing server of the shared table. In addition to most options supported in delta streaming,
+there are two additional options/spark configs.
+
+- spark config **spark.delta.sharing.streaming.queryTableVersionIntervalSeconds**: DeltaSharingSource
+leverages [getTableVersion](#query-table-version) rpc to check whether there are new data available
+to consume. In order to reduce the traffic burden to the delta sharing server, there's a minimum 30
+seconds interval between two getTableVersion rpcs to the delta sharing server. Though, if you are ok
+with less freshness on the data and want to reduce the traffic to the server, you can set this 
+config to a larger number, for example: 60s or 120s.  
```

Comment:
Consider adding that if this is set to less than 30, it will be ignored... right?